src/docker/Dockerfile

```diff
@@ -119,6 +120,16 @@ RUN wget -O /usr/local/bin/gomplate "https://github.com/hairyhenderson/gomplate/
   && chmod +x /usr/local/bin/gomplate \
   && rm -rf /var/tmp/* /tmp/* /var/tmp/.???* /tmp/.???*
 
+ # Install Podman. Get versions from https://github.com/hairyhenderson/gomplate/releases
+ARG PODMAN_VERSION="3.0.1-2"
+ENV PODMAN_VERSION=${PODMAN_VERSION}
+RUN dnf -y module disable container-tools
+RUN dnf -y install 'dnf-command(copr)'
+RUN curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:testing.repo https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/testing/CentOS_8/devel:kubic:libcontainers:testing.repo
+RUN dnf -y --refresh install podman
```

Comment:
- Can this happen as the `anvil` user? I think this needs to be moved up where `root` is doing stuff.
- What does `--refresh` do?
- We run a big `dnf install` towards the top of the dockerfile. This should be added to that

Reply:
hey, so Im having an issue with this.  I swtiched my curl to wget, based on Brad's comment.  Now, I have an issue putting the podman in the same dnf as the wget.  I need the wget to run first to stage the repo for the podman to install.

Reply:
Got it, that makes sense. Is there anything else that is needed first? We could do 2 `dnf install`s then, one for dependencies, then another for everything else.

Reply:
let me test that, and see if that will work with this new approach.